Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'aamva', github: '18F/identity-aamva-api-client-gem', tag: 'v3.4.1'
+gem 'aamva', '>= 3.5.0', github: '18F/identity-aamva-api-client-gem', branch: 'margolis-transaction-id'
```

Comment:
I can clean this up once https://github.com/18F/identity-aamva-api-client-gem/pull/35 merges